kolibri/plugins/learn/assets/src/views/ExploreLibrariesPage/LibraryItem.vue

```diff
@@ -0,0 +1,177 @@
+<template>
+
+  <KGrid
+    :debug="false"
+    class="library-item"
+  >
+    <div :class="{ 'library-header': true, 'library-header-sm': windowIsSmall }">
+      <KGridItem
+        :layout12="{ span: 10 }"
+        :layout8="{ span: 6 }"
+        :layout4="{ span: 4 }"
+      >
+        <div>
+          <h2 class="device-name">
+            <KIcon
+              v-if="deviceIcon"
+              :icon="deviceIcon"
+              class="device-icon"
+            />
+            <span>{{ deviceName }}</span>
+            <KIconButton
+              :icon="pinIcon"
+              appearance="flat-button"
+            />
+          </h2>
+          <p
+            v-if="showDescription"
+            class="device-description"
+          >
+            {{ $tr('channels', { channels: totalChannels }) }}
+          </p>
+        </div>
+      </KGridItem>
+      <KGridItem
+        :layout12="{ span: 2, alignment: 'right' }"
+        :layout8="{ span: 2, alignment: 'right' }"
+        :layout4="{ span: 4, alignment: 'right' }"
+      >
+        <KButton
+          :text="coreString('explore')"
+          :primary="false"
+        />
+      </KGridItem>
+    </div>
+    <div class="library-channels">
+      <KGridItem
+        v-for="channel in channels"
+        :key="channel.id"
+        :layout12="{ span: 3 }"
+        :layout8="{ span: 4 }"
+        :layout4="{ span: 4 }"
+      >
+        <ChannelCard
+          :title="channel.name"
+          :tagline="channel.description"
```

Comment:
One slight tweak here - in the other place where we use ChannelCard, we first try to display the channel tagline, and then the channel description if the channel tagline is null:
```
:tagline="content.tagline || content.description"
```
https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/learn/assets/src/views/ChannelCardGroupGrid.vue#L13

Reply:
This has been updated